gui/packages/desktop/src/renderer/components/Settings.tsx

```diff
@@ -85,33 +95,37 @@ export default class Settings extends Component<IProps> {
     const isOutOfTime = expiry ? expiry.hasExpired() : false;
     const formattedExpiry = expiry ? expiry.remainingTime().toUpperCase() : '';
 
+    // TRANSLATORS: OUT OF TIME
+    const outOfTimeMessage = pgettext('settings-view', 'account-cell-out-of-time-subtext');
```

Comment:
So with this solution the developers need to come up with a unique identifier for each string? For the website I know the original text (English) is used as the actual lookup key for the translation. Is that not possible using this tool?